# list of joint names  
std_msgs/String[] joint_names
---
# list of joints axes 
joint_prediction/JointAxis[] joints_axes